tests/searchlight/test_searchlight.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+from mpi4py import MPI
+import numpy as np
+import brainiak.searchlight.searchlight
+
+def test_create_searchlight():
+  sl = brainiak.searchlight.searchlight.Searchlight()
+  assert sl, "Invalid searchlight instance"
+
+def test_simple_searchlight():
+
+  def fn(a, mask):
+    return np.mean(a[0]) + np.mean(a[1])
+
+  M=5
+  N=7
+  K=12
+  D=10
+  R=1
+
+  data = None
+  mask = None
+  if MPI.COMM_WORLD.Get_rank() == 0:
+    data = [0.5*np.ones((D,M,N,K)), 0.5*np.ones((D,M,N,K))]
+    mask = np.ones((M,N,K))
+
+  sl = brainiak.searchlight.searchlight.Searchlight()
+  output = sl.run(data, mask, fn, R)
+
```

Comment:
Please add invalid (basic) tests as well.
Like one subject in data, radius = 0, null function, etc.